src/main/java/com/epam/dto/trainer/TrainerRegistrationRequest.java

```diff
@@ -1,10 +1,9 @@
 package com.epam.dto.trainer;
 
+import jakarta.validation.constraints.NotNull;
```

Comment:
I suggest to add more validations for the "specialization" string.  It could be a business logic validation, because you can easily miss a specialization from the list of correct specializations, right?